### ISSUE TITLE
[Idea] 

### ISSUE THREAD

**Issue by Freddy337** [MCCTeam/Minecraft-Console-Client#2578]
### Prerequisites

- [X] I have read and understood the [user manual](https://mccteam.github.io/guide/)
- [X] I made sure I am running the latest [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest)
- [X] I tried to [look for similar feature requests](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one

### Console Client Version

Latest

### Describe your problem

basically i dont know if its implemented but maybe add support to install mods on the client so we can join modded servers. if it is implemented please tell me how to do it. thanks

### Device

None

### Operating System

None

**Comment by milutinke** [comment]
Read https://github.com/MCCTeam/Minecraft-Console-Client/issues/2354
